controllers/upgrade/upgradeExecutor.go

```diff
@@ -114,29 +88,71 @@ func (ue *upgradeExecutor) Upgrade(ctx context.Context, toDoguResource *k8sv1.Do
 	}
 
 	ue.normalEventf(toDoguResource, "Extracting optional custom K8s resources...")
-	var customK8sResources map[string]string
-	customK8sResources, err = extractCustomK8sResources(ctx, ue.fileExtractor, toDoguResource, toDogu)
+	execPod, err := ue.execPodFactory.NewExecPod(util.ExecPodVolumeModeUpgrade, toDoguResource, toDogu)
 	if err != nil {
 		return err
 	}
+	err = execPod.Create(ctx)
+	if err != nil {
+		return err
+	}
+	defer execPod.Delete(ctx)
```

Comment:
🚫 **[checkstyle]** <errcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `execPod.Delete` is not checked